update on certificate errors (#21)
rstudio/apis-plumber: 07-hosting.Rmd

```
sudo npm install -g pm2
```

> If you find errors like SSL Error: CERT_UNTRUSTED while using npm command then to bypass https we use below commands:

Reviewer: ```suggestion
If you find errors like `SSL Error: CERT_UNTRUSTED` while using `npm` command, you can bypass the ssl error using:
```

```
npm config set strict-ssl false
```

> or set the registry URL from https or http like below: use

Reviewer: ```suggestion
or set the registry URL from `https://` to `http://`:
```
